test/integration/utils.ts

```diff
@@ -14,7 +16,7 @@ const chai = require("chai");
 global.expect = chai.expect;
 chai.Should();
 
-export const LATEST_ARC_VERSION = "0.1.1-rc.21";
+export const LATEST_ARC_VERSION = require("@daostack/test-env-experimental/package.json")["dependencies"]["@daostack/migration-experimental"];
```

Comment:
migration-experimental version is not the same as arc version. 

it should be `require("@daostack/migration-experimental/package.json")["dependencies"]["@daostack/arc-experimental"];`

Reply:
Right, in the other coment it was written `test-env-experimental`
I'll fix it.